### PR TITLE
chore(frontend): improve meta description

### PR DESCRIPTION
**Pull request by warnerHurtado**
### What does this PR do?

- resolves #424

### Steps to test
1. See the changes in homepage

#### CheckList
- [ ] Follow proper Markdown format
- [ ] The content is adequate
- [ ] The content is available in both english and spanish
- [ ] I Ran a spell check

### CODE REVIEW

File: src/pages/index.js [hunk 1]
```diff
@@ -183,49 +183,49 @@ const CardsSection = () => {
       <Grid
         container
         direction="row"
-        justify="center"
+        justifyContent="center"
```
Comment (warnerHurtado) [inline src/pages/index.js:186]: This solve the console error

File: src/pages/index.js [hunk 2]
```diff
@@ -183,49 +183,49 @@ const CardsSection = () => {
...
         alignItems="center"
         spacing={2}
       >
       <Grid item xs={12} sm={12} md={4}>
         <CustomCard 
           title="Guias generales"
-          body= "Guías para los desarrolladores sobre como programar en código abierto."
+          body= "Guías generales para el desarrollo en código abierto."
```
Comment (warnerHurtado) [inline src/pages/index.js:193]: I changed some descriptions

File: src/pages/index.js [hunk 3]
```diff
@@ -183,49 +183,49 @@ const CardsSection = () => {
...
-          body= "Conjunto de información necesaria para aprender sobre el protocolo EOSIO."
+          body= "Recursos y información para el aprendizaje del protocolo EOSIO."
           href= "/docs/aprender-eosio/protocolo-eosio"
           img="/img/cards-icons/icons-eosio.svg"></CustomCard>
       </Grid>
       <Grid item xs={12} sm={12} md={4}>
         <CustomCard 
           title="Tutoriales"
           body= "Información de apoyo necesaria para realizar la instalación y la ejecución de procesos. "
-          href= "docs/tutoriales/como-iniciar"
```
Comment (warnerHurtado) [inline src/pages/index.js:208]: This link didn't exist, so, I changed it.

**Review by xavier506 — APPROVED** [review]
LGTM

**Pull request merged**